components/eamxx/src/share/io/scorpio_output.cpp

```diff
@@ -638,6 +644,13 @@ run (const std::string& filename,
       grid_write_data_array(filename,name,view_host.data(),view_host.size());
     }
   }
+  auto func_finish = std::chrono::steady_clock::now();
+  if (is_write_step) {
+    if (m_atm_logger) {
+      auto duration = std::chrono::duration_cast<std::chrono::milliseconds>(func_finish - func_start)/1000.0;
+      m_atm_logger->info("[EAMxx::scorpio_output] Writing variables to file:\n\t " + filename + " ...done! (Elapsed time = " + std::to_string(duration.count()) +" seconds)\n");
```

Comment:
To be fair, this is not just measuring the time it takes to write to file, but also the time it took to remap and do the accumulation. You could either always print this (regardless of the value of `is_write_step`) or maybe do a more accurate clocking of the pio write function.

Reply:
Good point.  I could probably just assign `func_start` before the grid_write_data_array call to capture just the time in writing the file to more accurately capture the time in pio.

Reply:
Yeah, that's why I said I didn't know what you wanted to capture. PIO already logs the time it takes to write, but I guess this would give you a sense of how much each write costs (e.g., restart vs output). Up to you.